gostream/codec/h264/ffmpeg/avutil/avutil.go

```diff
@@ -0,0 +1,102 @@
+// Package avutil is a thin wrapper around FFmpeg/release4.4.
+// See: https://github.com/FFmpeg/FFmpeg/tree/release/4.4
+package avutil
+
+//#cgo CFLAGS: -Wno-deprecated-declarations -I${SRCDIR}/../ffmpeg/include
+//#cgo LDFLAGS: -L${SRCDIR}/../ffmpeg/lib -lavformat -lavcodec -lavutil -lavdevice -lavfilter -lswresample -lswscale -lm -lX11
```

Comment:
```suggestion
//#cgo LDFLAGS: -L${SRCDIR}/../ffmpeg/lib -lavutil -lswresample
```
I don't think you need all those other libraries.

Reply:
Done.